PeerioTechnologies/peerio-desktop
# feat: drag and drop between folders

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/7210/desktop-drag-drop-between-folders
https://app.clubhouse.io/peerio/story/8507/drag-and-drop-flows
https://app.clubhouse.io/peerio/story/7394/document-drag-and-drop

Some extra info about design choices made: https://app.clubhouse.io/peerio/story/9510/shared-folders-plus-drag-drop-is-confusing

Copy PR: https://github.com/PeerioTechnologies/peerio-icebear/pull/248

#### Testing instructions  

- Drag a single item into a non-shared folder
- Drag multiple items into a non-shared folder
- Repeat both of the above with shared folders (should display a confirmation if the user hasn't selected to never warn about it)

~~~KNOWN ISSUES: dragging a file from the desktop into the Files pane might break after doing it once -- seems to be a conflict between the global drag and drop context and the files-specific one. To be verified~~~
Should be fixed.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
